.github/workflows/main.yml

```diff
@@ -102,3 +102,15 @@ jobs:
         run: |
           sleep 5
           python scripts/api_e2e.py 8002
+
+# TODO add a step to run db migration, needs DATABASE_URL through secrets ?
+
+#  db-migration:
+#    runs-on: ubuntu-latest
+#    needs: end-to-end
+#    steps:
+#      - name: Run database migration
+#        env:
+#          DATABASE_URL: ${{ secrets.DATABASE_URL }}
+#        run: |
+#          alembic upgrade head
```

Comment:
We just need to test that this would work on an existing DB. Perhaps we could check with our unittests

Reply:
I stil need to check the pytest-alembic or equivalent, anyway we will need to create a specific github action "on push branch master" (and not on pull request): we want to run the migration only once the PR is merged right ?

Reply:
Correct, so I will add a CI job in another PR I guess

Reply:
I opened #140 about this